hypervisor/common/sched_bvt.c

```diff
@@ -30,17 +30,49 @@ struct sched_bvt_data {
 	uint64_t start_tsc;
 };
 
-static int sched_bvt_init(__unused struct sched_control *ctl)
+static void sched_tick_handler(__unused void *param)
 {
-	return 0;
 }
 
-static void sched_bvt_deinit(__unused struct sched_control *ctl)
+static int sched_bvt_init(struct sched_control *ctl)
 {
+	struct sched_bvt_control *bvt_ctl = &per_cpu(sched_bvt_ctl, ctl->pcpu_id);
+	uint64_t tick_period = BVT_MCU_MS * CYCLES_PER_MS;
+	int ret = 0;
+
+	ASSERT(get_pcpu_id() == ctl->pcpu_id, "Init scheduler on wrong CPU!");
```

Comment:
add @pre to make sure ASSERT never hits

Reply:
Will change to @pre.

Reply:
both @pre and ASSERT is preferred. The former one is for FuSa, and the later for debug